common/flavorization/00_CTP.txt

```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	priority = 40
+	cultures = { russian  }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+title_prince_female_russian_empire = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = empire
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+king_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+king_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+kingdom_russian = {
+	type = title
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+title_prince_male_russian_kingdom = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = kingdom
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+title_prince_female_russian_kingdom = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = kingdom
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+queen_mother_russian = {
+	type = character
+	special = queen_mother
+	tier = kingdom
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government }
+	top_liege = no
+}
+duke_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 28
+	governments = { feudal_government }
+	cultures = { russian }
+}
+duke_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 28
+	governments = { feudal_government }
+	cultures = { russian }
+}
+duchy_feudal_russian = {
+	type = title
+	tier = duchy
+	priority = 28
+	governments = { feudal_government }
+	cultures = { russian }
+}
+count_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 18
+	cultures = { russian }
+	governments = { feudal_government }
+	top_liege = no
+}
+count_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 18
+	cultures = { russian }
+	governments = { feudal_government }
+	top_liege = no
+}
+county_russian = {
+	type = title
+	tier = county
+	priority = 18
+	cultures = { russian  }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+baron_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { russian ilmenian severian volhynian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+baron_female_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { russian ilmenian severian volhynian }
```

Comment:
I think using the whole cultural group, instead of listing all Induvidual cultures should be used for readability and debugging puporses